ci-operator/step-registry/osd-ccs/fleet-manager/sc/provision/osd-ccs-fleet-manager-sc-provision-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: osd-ccs-fleet-manager-sc-provision
+  from_image:
+    namespace: ci
+    name: cli-ocm
+    tag: latest
+  grace_period: 3h10m
+  commands: osd-ccs-fleet-manager-sc-provision-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 300Mi
+  env:
+  - name: OCM_LOGIN_ENV
+    default: "staging"
+    documentation: The environment for ocm login. The supported values are [production, staging].
+  - name: CLOUD_PROVIDER_REGION
+    default: ""
+    documentation: Set a region for the ocm cluster creation. If not specified, a cluster_profile appropriate setting will be used.
+  documentation: |-
+    Create sc using ocm cli.
```

Comment:
Do not use the abbreviation word `sc` in the description. And the description should be like 'Using the OSD Fleet Manager api endpoint to provision the service cluster, and a default management cluster will be provisioned together. '